pages/news-and-views/_slug/index.vue

```diff
@@ -12,37 +14,57 @@ export default {
 		const linkList = store.state.posts.map(({ slug }) => slug)
 		return linkList.includes(params.slug)
 	},
-	head () {
-		return {
-			title: this.Title
-		}
-	},
+	// head () {
+	// 	return {
+	// 		title: this.Title
+	// 	}
+	// },
 	components: {
 		PostArticle
 	},
 	data () {
 		return {
 		}
 	},
-	computed: {
-		links () {
-			return this.$store.state.posts.map(({ slug }) => slug)
-		},
-		linkNames () {
-			return this.links.map(x => x.replace(/-/g, " "))
-		},
-		Title () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).title
-		},
-		Picture () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).picture
-		},
-		Content () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).content
-		},
-		Tags () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).tags
-		}
+	// computed: {
+	// 	links () {
+	// 		return this.$store.state.posts.map(({ slug }) => slug)
+	// 	},
+	// 	linkNames () {
+	// 		return this.links.map(x => x.replace(/-/g, " "))
+	// 	},
+	// 	Title () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).title
+	// 	},
+	// 	Picture () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).picture
+	// 	},
+	// 	Content () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).content
+	// 	},
+	// 	Tags () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).tags
+	// 	}
+	// },
+	asyncData (context) {
+		return axios.get(`${Config.wpDomain}${Config.apiBase}posts`).then((response) => {
+			const res = response.data.filter(x => x.slug === context.params.slug)[0]
```

Comment:
Using a meaningful variable would be helpful for others to understand the code. Could you rename the var name "x" to "post" or other names that are meaningful. Thanks.